king_phisher/client/windows/compare_campaigns.py

```diff
@@ -0,0 +1,155 @@
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above
+#    copyright notice, this list of conditions and the following disclaimer
+#    in the documentation and/or other materials provided with the
+#    distribution.
+#  * Neither the name of the project nor the names of its
+#    contributors may be used to endorse or promote products derived from
+#    this software without specific prior written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+#  "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+#  LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+#  A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+#  OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+#  SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+#  LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+#  DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+#  THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+#  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+#  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+
+from king_phisher import its
+from king_phisher import utilities
+from king_phisher.constants import ColorHexCode
+from king_phisher.client import gui_utilities
+from king_phisher.client.widget import managers
+from king_phisher.client.graphs import CampaignCompGraph
+
+from gi.repository import Gtk
+from gi.repository import Gdk
+
+class CampaignCompWindow(gui_utilities.GladeGObject):
+	"""
+	The window which allows the user to select campaigns and compare the data
+	using graphical representation.
+	"""
+	dependencies = gui_utilities.GladeDependencies(
+		children=(
+			'treeview_campaigns',
+			'scrolledwindow',
+			'stackswitcher',
+			'box_stack',
+			'stack_1'
+		),
+	)
+	top_gobject = 'window'
+	def __init__(self, *args, **kwargs):
+		super(CampaignCompWindow, self).__init__(*args, **kwargs)
+		treeview = self.gobjects['treeview_campaigns']
+		self.campaigns_enabled = list()
+		tvm = managers.TreeViewManager(
+			treeview,
+			cb_refresh=self.load_campaigns
+		)
+		toggle_renderer = Gtk.CellRendererToggle()
+		toggle_renderer.connect('toggled', self.signal_renderer_toggled)
+		stack_switcher = self.gobjects['stackswitcher']
+		stack_switcher.connect('button-release-event', self.show_options)
+		self.box_stack = self.gobjects['box_stack']
+		self.stack = self.gobjects['stack_1']
+		self.signal_ready = False
+		b = Gtk.CellRendererText()
+		tvm.set_column_titles(
+			('Compare', 'Name', 'Company', 'Type', 'Created By', 'Creation Date', 'Expiration'),
+			column_offset=1,
+			renderers=(toggle_renderer, b, b, b, b, b, b)
+		)
+		self._model = Gtk.ListStore(str, bool, str, str, str, str, str, str, Gdk.RGBA, Gdk.RGBA, str)
```

Comment:
The last three columns here are unnecessary because we don't need to highlight any rows or provide tooltips.